content/en/methods/push.md

```diff
@@ -45,14 +45,15 @@ Add a Web Push API subscription to receive notifications. Each access token can
 3.3.0 - added `data[alerts][status]`\
 3.4.0 - added `data[policy]`\
 3.5.0 - added `data[alerts][update]` and `data[alerts][admin.sign_up]`\
-4.0.0 - added `data[alerts][admin.report]`
+4.0.0 - added `data[alerts][admin.report]`\
+4.3.0 - added stricter request parameter validation, invalid endpoint URLs and subscription keys will now result in an error, previously these would be accepted, but silently fail.
```

Comment:
run-on sentence

Reply:
What would be your suggested fix? This to me looks fine when I only really have a single sentence to use?